### DISCUSSION TITLE
How to add a blog article to the netbox site?

### DISCUSSION THREAD

**Discussion by misch42** [netbox-community/netbox#14605] [Q&A]
### Change Type

Addition

### Area

Features

### Proposed Changes

I wrote some blog articles about netbox on my webseite sys4.de/blog. How can I add blog articles to the https://netbox.dev/blog/ site?

Michael Schwartzkopff

**Comment by jeffgdotorg** [comment]
I'm working with @misch42 and some colleagues in [a Slack thread](https://netdev-community.slack.com/archives/C01P0FRSXRV/p1703414259390749) to move this forward. Since that history will expire, I'm linking to [Michael's `autonetz` repo](https://github.com/misch42/autonetz/blob/main/doc/autonetz.adoc) where the blog post content is currently under development.
